### ISSUE TITLE
[Screen Readers - LxP Assist]: Visually appearing heading multiple text is not defined as heading present in the 'Lxp Assist' Flyout.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4829]
### **PWD Impact:**
Users who rely on Screen reader will be impacted as Heading tag is not provided for multiple text which is visually looks like heading and screen readers are not narrating the heading information for the multiple text.

### **Repro Steps:**
Step 1: Open the above URL in Edge Chromium/Chrome Browser.
Step 2: 'Enterprise Skills Initiative' page will appear.> Navigate to the 'Click to open Copilot' button and click on it. > 'LXP Assist' flyout will appear >Navigate inside the flyout using arrow key and shortcut key 'h'.
Step 3: Observe whether Visually appearing heading multiple text is defined as heading present in the 'Lxp Assist' Flyout or not.

### **Actual Result:**
Visually appearing heading multiple text is not defined as heading present in the 'Lxp Assist' Flyout.

### **Expected Result:**
Visually appearing heading multiple text  should be defined as heading present in the 'Lxp Assist' Flyout.

### **MAS Reference Link:**
[MAS 1.3.1 - Info and Relationships](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7d)

### **WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
### **Test Environment:**
OS: Windows 11 22H2 (OS Build 22621.1992)
Browser: Microsoft Edge Version (114.0.1823.82), Chrome (114.0.5735.199)
URL: [https://ui.int.esi.microsoft.com/](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fui.int.esi.microsoft.com%2f&data=05%7c01%7cv-rmounika%40microsoft.com%7c4ed10265b9a246cbbba108db84c198c6%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638249739385137411%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=NWSDitB8DvvJww0%2fMAZw1XoxXQrpuW14b89LtmxoRpU%3D&reserved=0)

**Comment by pgopinadhu** [comment]
Test Environment:
OS: Windows 11 22H2 (OS Build 22621.1992)
Browser: Microsoft Edge Version (114.0.1823.82), Chrome (114.0.5735.199)
URL: [https://ui.int.esi.microsoft.com/](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fui.int.esi.microsoft.com%2f&data=05%7c01%7cv-rmounika%40microsoft.com%7c4ed10265b9a246cbbba108db84c198c6%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638249739385137411%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=NWSDitB8DvvJww0%2fMAZw1XoxXQrpuW14b89LtmxoRpU%3D&reserved=0)

**Comment by pgopinadhu** [comment]
#HCL; #Accessibility; #Benchmark; #A11yMAS; #A11ySev2; #MAS1.3.1; #Win11-Edge(Chromium); #WWL:Bug1657333; #WWL;
#FeatureBotframeworkwebchat-Apr23; Win11-Chrome; #AILimited;

**Comment by compulim** [comment]
For the first issue, the bot author need to add a title in their Adaptive Cards.

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/313de29d-8eec-4340-a9fb-11cfe03b3e54)

For the second issue, the visual highlighted is not part of Web Chat.

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/2b307b15-647e-4446-849e-4c0cdf8a0473)

Closing as this is not related to Web Chat.
